modules/osdk-building-ansible-operator.adoc

```diff
@@ -0,0 +1,377 @@
+// Module included in the following assemblies:
+//
+// * operators/osdk-ansible.adoc
+
+[id='osdk-building-ansible-operator-{context}']
+= Building an Ansible-based Operator using the Operator SDK
+
+This procedure walks through an example of building a simple Memcached Operator
+powered by Ansible playbooks and modules using tools and libraries provided by
+the Operator SDK.
+
+.Prerequisites
+
+- Operator SDK CLI installed on the development workstation
+- Access to a Kubernetes-based cluster v1.11.3+ (for example {product-title} 4.0)
+using an account with `cluster-admin` permissions
+- link:https://kubernetes.io/docs/tasks/tools/install-kubectl/[`kubectl`] v1.11.3+
+(can alternatively use `oc`)
+- link:https://docs.ansible.com/ansible/latest/index.html[`ansible`] v2.6.0+
+- link:https://ansible-runner.readthedocs.io/en/latest/install.html[`ansible-runner`] v1.1.0+
+- link:https://github.com/ansible/ansible-runner-http[`ansible-runner-http`] v1.0.0+
+
+.Procedure
+
+. *Create a new Operator project,* either namespace-scoped or cluster-scoped,
```

Comment:
Do you guys normally bold portions of procedure steps? I remember this coming up a few years back and it was decided not to use bold in steps, but I don't remember if that was just an EAP guideline, or CCS-wide.

Reply:
I have started doing it in 4.0 topics because I always liked when @chrisnegus did it... but... I don't remember anymore what conversations we've had recently about it. I believe it came up when I was gunning for some CSS changes to our markup last year (which are now in-place for 4.0 docs), and being able to do this was one of the (many) driving factors (for me).

FWIW, I don't recall a CCS-wide hand-slap. Probably worth a bring-up in our team mtg.